src/roinput.rs

```diff
@@ -0,0 +1,658 @@
+use super::*;
+
+use algebra::{
+    PrimeField, // for size_in_bits()
+};
+
+use bitvec::{
+    prelude::*,
+    view::AsBits,
+};
```

Comment:
try to not have line breaks between `use` calls

Reply:
Done!